tests/utils_tests/test_datastructures.py

```diff
@@ -34,6 +35,12 @@ def test_discard(self):
         s.discard(2)
         self.assertEqual(len(s), 1)
 
+    def test_reversed(self):
+        s = OrderedSet([1, 2, 3])
+        self.assertEqual(list(reversed(s)), [3, 2, 1])
+        # Just checking that it's some iterator
```

Comment:
Since you're checking against `collections.abc.Iterator`, you can leave this comment out. (I was suggesting a comment like that only if you were comparing it against the concrete type `type(reversed({}))`.)

Also, you can use `assertIsInstance()` below: https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertIsInstance